src/components/structures/RoomView.tsx

```diff
@@ -928,7 +928,7 @@ export class RoomView extends React.Component<IRoomProps, IRoomState> {
             }
 
             case "scroll_to_bottom":
-                if (payload.timelineRenderingType === this.context.timelineRenderingType) {
+                if (payload.timelineRenderingType === this.state.timelineRenderingType) {
```

Comment:
```suggestion
                if (payload.timelineRenderingType === TimelineRenderingType.Room {
```

I believe this would be more appropriate in this situation. In `RoomView` we're only interested in having the actions from the room context.
Not entirely sure why we store the timeline rendering type in state there, as it will never change. Will review this piece of code